src/More/LanguageSelection.tsx

```diff
@@ -2,11 +2,13 @@ import React from "react"
 import { FlatList, View, StyleSheet, TouchableHighlight } from "react-native"
 import { useTranslation } from "react-i18next"
 import { useNavigation } from "@react-navigation/native"
+import { useStatusBarEffect } from "../navigation"
 
 import { getLocaleList, setUserLocaleOverride } from "../locales/languages"
 import { GlobalText } from "../components/GlobalText"
 
 import { Colors, Spacing, Typography } from "../styles"
+import { SafeAreaView } from "react-native-safe-area-context"
```

Comment:
what do you think about keeping this dependency at the top with the other third party dependencies?

Reply:
This can actually come directly from "react-native" @emilianojuan